test_file

```diff
@@ -0,0 +1 @@
+parent foo
```

Comment:
It might be better to make the test directory deeper such that this test file doesn't end up at the root of the project.

Reply:
Yep, good call - have tidied that up a bit in the next commit.